src/firmware/boards/robot_stm32h7/io/drivetrain.c

```diff
@@ -46,3 +46,72 @@ void io_drivetrain_applyForceBackRightWheel(float force_newtons)
     assert(initialized);
     io_drivetrain_unit_applyForce(_back_right_drive_unit, force_newtons);
 }
+
+float io_drivetrain_getFrontLeftRpm(void)
+{
+    // TODO (#2098) implement reading rpm
+    return 100.0f;
+}
+
+float io_drivetrain_getFrontRightRpm(void)
+{
+    // TODO (#2098) implement reading rpm
+    return 100.0f;
+}
+
+float io_drivetrain_getBackLeftRpm(void)
+{
+    // TODO (#2098) implement reading rpm
+    return 100.0f;
+}
+
+float io_drivetrain_getBackRightRpm(void)
+{
+    // TODO (#2098) implement reading rpm
+    return 100.0f;
+}
+
+void io_drivetrain_brakeFrontLeft(void)
+{
+    io_drivetrain_unit_applyForce(_front_left_drive_unit, 0.0f);
+    assert(initialized);
```

Comment:
should this assertion be before we apply force?

Reply:
Also possibly out-of-scope, but we should add a comment to `initialized` at the declaration explaining what it is

Reply:
ah yeah this wasn't really doing anything :sweat_smile: 

added a comment as well